examples/nvme/perf/perf.c

```diff
@@ -475,6 +480,21 @@ submit_single_io(struct ns_worker_ctx *ns_ctx)
 	uint64_t		offset_in_ios;
 	int			rc;
 	struct ns_entry		*entry = ns_ctx->entry;
+	uint64_t current_time;
+
+	if (ns_ctx->io_limit) {
+		current_time = spdk_get_ticks();
+		if ((current_time - ns_ctx->io_limit_time) > g_tsc_rate) {
+			ns_ctx->io_limit_count = 0;
```

Comment:
suggest to send io_limit count to something like io_submited_cnt

Reply:
done